projects/upgrade-executor/test/trasnfer-ownership.test.ts

```diff
@@ -192,10 +220,27 @@ describe("Transfer Ownership", function () {
     //vote for proposal
     const voteTx = await governor.castVote(proposalId, VoteType.For)
 
-    const voteReceipt = await voteTx.wait(37000)
```

Comment:
Used hardhat_mine instead of .wait(blocks) to advance blocks without having to wait in real-time, bypassing the timeout errors.